assets/src/edit-story/output/utils/styles.js

```diff
@@ -48,6 +48,15 @@ function CustomStyles() {
                 overflow: visible;
               }
 
+              @media (max-aspect-ratio: 9 / 16)  {
+                @media (min-aspect-ratio: 1 / 2) {
+                  amp-story-grid-layer#amp-story-grid-layer-bg,
+                  amp-story-grid-layer#amp-story-grid-layer-fullbleed-area {
+                    margin-top: calc((${FULLBLEED_RATIO} * 100% / 2) - ${SAFE_ZONE_HEIGHT}px);
```

Comment:
Since `SAFE_ZONE_HEIGHT` and `FULLBLEED_RATIO` are imported from elsewhere we won't notice here if they should change to something problematic. We're using `dangerouslySetInnerHTML` here, so it might be a good idea to sanitize them at the top of the function just to be safe - like for the other variables.

Reply:
Got it.